File: Server/mobile_o2o_project_local_server_share_231115/src/main/java/com/ssafy/cafe/model/dto/ReComment.java
package com.ssafy.cafe.model.dto;

public class ReComment {
	private Integer id;
	private Integer commentId;
	private Integer productId;
	private String comment;
	
	public ReComment() {
		
	}
	
	public ReComment(Integer productId, String comment) {
		this.productId = productId;
		this.comment = comment;
	}
	
	public Integer getId() {
		return id;
	}
	public void setId(Integer id) {
		this.id = id;
	}
	public Integer getCommentId() {
		return commentId;
	}
	public void setCommentId(Integer commentId) {
		this.commentId = commentId;
	}
	
	public Integer getProductId() {
		return productId;
	}

	public void setProductId(Integer productId) {
		this.productId = productId;
	}
	
	public String getComment() {
		return comment;
	}
	public void setComment(String comment) {
		this.comment = comment;
	}
	
}

File: Andrioid/Admin/app/src/main/java/com/ssafy/smartstore_jetpack/dto/ReComment.kt
package com.ssafy.smartstore_jetpack.dto

data class ReComment(
    val id: Int = -1,
    val commentId: Int = -1,
    var productId: Int = -1,
    var comment: String
){
    constructor(comment_id: Int, comment: String): this(0, comment_id, -1, comment)
    constructor(id: Int, commentId: Int, comment: String): this(id, commentId, -1, comment)
}
